XTS/WS-C/pom.xml

```diff
@@ -149,6 +149,21 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>[17,)</jdk>
+      </activation>
+      <properties>
+        <jvm.args.modular>
+          --add-modules=java.se
```

Comment:
Do you know why the JDK 11 profile didn't need this module adding before?

Reply:
Same question for the XTS/localjunit/pom.xml, rts/at/tx/pom.xml and rts/lra/test/pom.xml?

Reply:
Sorry, I was not clear. This is not quite the same as the queries I was having about overriding. 

If you look in https://github.com/jbosstm/narayana/blob/98e4306577a3b8a8857ae16d33d87af1e5a4bb52/XTS/WS-C/pom.xml you can see that with the changes in this pull request, the result is that the java.se module is added for JDK 17 (but it's not added in JDK 11). This is different to the other situations where it seems the pull is adding for JDK 17 but wasn't present for the existing JDK 11 build (for instance https://github.com/jbosstm/narayana/blob/98e4306577a3b8a8857ae16d33d87af1e5a4bb52/ArjunaJTA/cdi/pom.xml#L298). Please can you help to explain the reason you think that the JDK 11 build didn't need it (before) but you it's added here JDK 17?

Reply:
@tomjenkinson here I think, the modules that need access to `java.base/java.lang` (reflective access) and some other packages need to add `--add-modules=java.se`

Reply:
Thanks @mayankkunwar